packages/interface/src/getHoverInformation.js

```diff
@@ -0,0 +1,186 @@
+/**
+ *  Copyright (c) Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the license found in the
+ *  LICENSE file in the root directory of this source tree.
```

Comment:
I have specified the wrong licence I think. `graphql/codemirror-graphql` is under BSD, which requires redistributions (including those with modifications) to follow the same.

Thoughts?

Reply:
We should change all license info to MIT (I believe we had a very interesting series of events because of this 😃). I haven't changed licensing info for `graphql-language-service` and `codemirror-graphql` just yet.

Reply:
That would be a worthwhile effort.